databricks/koalas/tests/test_series.py

```diff
@@ -155,6 +155,18 @@ def test_isin(self):
         with self.assertRaisesRegex(TypeError, msg):
             ds.isin(1)
 
+    def test_fillna(self):
+        ps = pd.Series([np.nan, 2, 3, 4, np.nan, 6], name='x')
+        ks = koalas.from_pandas(ps)
+
+        self.assert_eq(ks.fillna(0), ps.fillna(0))
+        self.assert_eq(ks.fillna(0, inplace=True), ps.fillna(0, inplace=True))
```

Comment:
i don't think this test case is correct. in both cases inpalce=True returns nothing. We need to compare the ks.

Also we probably need to make a copy of it. Otherwise you pollute the following "ks" because ks has been changed.

Reply:
Agree! I'll fix it.